tests/integration/api_v2/rpc/blocks.test.js

```diff
@@ -175,11 +175,88 @@ describe('Method get.blocks', () => {
 			const response = await getBlocks({ timestamp: `${from}:${to}` });
 			expect(response).toMap(jsonRpcEnvelopeSchema);
 			const { result } = response;
-			expect(result.data[0]).toMap(blockSchemaVersion5);
+			expect(result.data).toBeInstanceOf(Array);
+			expect(result.data.length).toBeGreaterThanOrEqual(1);
+			expect(result.data.length).toBeLessThanOrEqual(10);
 			result.data.forEach((blockItem) => {
+				expect(blockItem).toMap(blockSchemaVersion5);
 				expect(blockItem.timestamp).toBeGreaterThanOrEqual(from);
 				expect(blockItem.timestamp).toBeLessThanOrEqual(to);
 			});
 		});
+
+		it('Blocks with from... timestamp -> ok', async () => {
+			const from = 0;
```

Comment:
Please consider using the `refBlock.timestamp` instead.
```suggestion
			const from = refBlock.timestamp - 100;
```